modules/client/src/main/kotlin/jp/co/soramitsu/iroha2/Iroha2Client.kt

```diff
@@ -235,28 +243,33 @@ open class Iroha2Client(
                 }
                 is TransactionRejectionReason.NotPermitted -> reason.notPermittedFail.reason
                 is TransactionRejectionReason.SignatureVerification -> reason.signatureVerificationFail.reason
-                is TransactionRejectionReason.UnexpectedGenesisAccountSignature -> "Genesis account can sign only transactions in the genesis block"
-                is TransactionRejectionReason.UnsatisfiedSignatureCondition -> reason.unsatisfiedSignatureConditionFail.reason
+                is TransactionRejectionReason.UnexpectedGenesisAccountSignature ->
+                    "Genesis account can sign only transactions in the genesis block"
+                is TransactionRejectionReason.UnsatisfiedSignatureCondition ->
+                    reason.unsatisfiedSignatureConditionFail.reason
+                else -> throw IllegalArgumentException(
```

Comment:
The branch `WasmExecution` is not covered. Please cover it. With this branch `else` will be unnecessary